src/sonic-yang-models/tests/yang_model_tests/tests_config/port.json

```diff
@@ -12,6 +12,7 @@
                         "mtu": 9000,
                         "pfc_asym": "off",
                         "name": "Ethernet8",
+                        "tpid": "0x8100",
```

Comment:
No need to update old tests.

Reply:
TPID will be present in all config DB after thei feature is checked in so making sure it is consistently present in all interface configs.